src/openMVG/calibration/bestImages.hpp

```diff
@@ -10,62 +10,64 @@ namespace calibration{
+                           const cv::Size& imageSize,
+                           const std::size_t calibGridSize,
+                           std::vector<std::vector<std::size_t> >& cellIndexesPerImage);
 
 /**
  * @brief This function counts the number of points in each cell of the grid.
  *
- * @param[in] imagesIndexes
- * @param[in] cellIndexesPerImage
- * @param[out] cellsWeight
- * @param[in] calibGridSize
+ * @param[in] imagesIndexes The images id of the sequence.
+ * @param[in] cellIndexesPerImage The id of the cell for each point of the image sequence.
+ * @param[in] calibGridSize The number of cells per edge.
+ * @param[out] cellsWeight The number of points for each cell id.
  */
 void computeCellsWeight(const std::vector<std::size_t>& imagesIndexes,
                         const std::vector<std::vector<std::size_t> >& cellIndexesPerImage,
-                        std::map<std::size_t, std::size_t>& cellsWeight, const std::size_t calibGridSize);
+                        const std::size_t calibGridSize,
+                        std::map<std::size_t, std::size_t>& cellsWeight);
 
 /**
  * @brief This function computes the score of each image.
  *
- * @param[out] imageScores
- * @param[in] remainingImagesIndexes
- * @param[in] cellIndexesPerImage
- * @param[in] cellsWeight
+ * @param[in] remainingImagesIndexes The ids of the images in which we pick up the maxCalibFrames with the best score.
+ * @param[in] cellIndexesPerImage The id of the cell for each point of the image sequence.
+ * @param[in] cellsWeight The number of points for each cell id.
+ * @param[out] imageScores The score of each image.
  */
-void computeImageScores(std::vector<std::pair<float, std::size_t> >& imageScores,
-                        const std::vector<std::size_t>& remainingImagesIndexes,
+void computeImageScores(const std::vector<std::size_t>& remainingImagesIndexes,
                         const std::vector<std::vector<std::size_t> >& cellIndexesPerImage,
-                        const std::map<std::size_t, std::size_t>& cellsWeight);
+                        const std::map<std::size_t, std::size_t>& cellsWeight,
+                        std::vector<std::pair<float, std::size_t> >& imageScores);
 
 /**
  * @brief This function selects the best images based on repartition in images of the calibration landmarks.
  *
- * @param[in] imagePoints
- * @param[in] imageSize
- * @param[in,out] remainingImagesIndexes
- * @param[in] maxCalibFrames
- * @param[in] validFrames
- * @param[out] calibImageScore
- * @param[out] calibInputFrames
- * @param[out] calibImagePoints
- * @param[in] calibGridSize
+ * @param[in] imagePoints Vector of detected points for each image
+ * @param[in] imageSize Image size (width, height)
+ * @param[in] maxCalibFrames Maximum number of images to used for the calibration. 
+ * @param[in] validFrames Store the id of the detected checkers
+ * @param[in] calibGridSize The number of cells per edge.
```

Comment:
edge may be confused with some computer stuff. I propose "The number of cells per each image dimension.